scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/vqfn.yaml

```diff
@@ -560,3 +560,62 @@ VQFN-32-1EP_5x5mm_P0.5mm_EP3.1x3.1mm:
   #pad_length_addition: 0.5
   #suffix: '_Pad{pad_x:.2f}x{pad_y:.2f}mm_HandSolder'
   #include_suffix_in_3dpath: 'False'
+
+VQFN-32-1EP_5x5mm_P0.5mm_EP3.5x3.5mm:
+  device_type: 'VQFN'
+  #manufacturer: 'man'
+  #part_number: 'mpn'
+  size_source: 'https://www.ftdichip.com/Support/Documents/TechnicalNotes/TN_166%20FTDI%20Example%20IC%20PCB%20Footprints.pdf#page=22'
+  ipc_class: 'qfn' # 'qfn_pull_back'
+  #ipc_density: 'least' #overwrite global value for this device.
+  # custom_name_format:
+  body_size_x:
+    nominal: 5
+    maximum: 5.1
+  body_size_y:
+    nominal: 5
+    maximum: 5.1
+  overall_height:
+    minimum: 0.8
+    nominal: 0.85
+    maximum: 0.9
+  lead_width:
+    minimum: 0.18
+    nominal: 0.25
+    maximum: 0.3
+  lead_len:
+    minimum: 0.35
+    nominal: 0.4
+    maximum: 0.45
+
+  EP_size_x:
+    minimum: 3.45
+    nominal: 3.5
+    maximum: 3.55
+  EP_size_y:
+    minimum: 3.45
+    nominal: 3.5
+    maximum: 3.55
+  # EP_paste_coverage: 0.65
+  EP_num_paste_pads: [2, 2]
+
+  thermal_vias:
+    count: [3, 3]
+    drill: 0.33
+    # min_annular_ring: 0.15
+    paste_via_clearance: 0.1
+    # EP_num_paste_pads: [2, 2]
+    # paste_between_vias: 1
+    # paste_rings_outside: 1
+    EP_paste_coverage: 0.7
+    grid: [1.2, 1.2]
```

Comment:
Is there a reason to manually enter the grid?

Reply:
No, this is also due to copy-pasting the footprint above.

Reply:
In this case remove the grid definition.

Reply:
Also removed this. The FP now looks like this:

![screenshot-2019-09-01-22:18:19](https://user-images.githubusercontent.com/1786438/64081818-6df85580-cd06-11e9-9d25-00711786f503.png)